packages/integration-sdk-runtime/src/execution/executeIntegration.ts

```diff
@@ -284,18 +284,13 @@ export async function executeWithContext<
         data: summary,
       });
 
-      context.logger.info(
```

Comment:
Alternatively, you could drop this to `.debug`. I agree that it's not useful for us though. Thanks for improving the `Undeclared types` log message alongside this change 🔥